search_commands.go

```diff
@@ -0,0 +1,1215 @@
+package redis
+
+import (
+	"context"
+	"strings"
+)
+
+type SearchCmdable interface {
+	FT_List(ctx context.Context) *StringSliceCmd
```

Comment:
Can you ping Adriano - I know we have no other way.. but this exposes this (specific) item, even though we may want to walk it back